core/util/google.py

```diff
@@ -1,52 +1,68 @@
 """
 OWASP Maryam!
-
 This program is free software: you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 any later version.
-
 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
-
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
 
 class main:
 	# framework = None
-	def __init__(self, q, limit=1, count=10):
+	def __init__(self, q, limit=1, count=10, mode='original'):
 		""" google.com search engine
-
 			q     : Query for search
 			limit : Number of pages
 			count : Number of results
 		"""
 		self.framework = main.framework
 		self.q = q
-		self.agent = 'Mozilla/5.0 (X11; Linux x86_64; rv:86.0) Gecko/20100101 Firefox/86.0'
+		self.mode = mode
```

Comment:
How can someone change the mode manually?

Reply:
I haven't got it.